_locales/pt_BR/messages.json

```diff
@@ -480,17 +619,11 @@
         "message": "Ocultar Shorts remixando alguns videos"
     },
     "Hide_sidebar": {
-        "message": "Ocultar barra lateral de recomendações "
-    },
-    "Hide_subscriptionsShorts": {
-        "message": "Ocultar Shorts das inscrições"
+        "message": "Ocultar barra lateral"
```

Comment:
this one is by mistake or misunderstanding?

Reply:
String is correct, previous translation was wrong

Reply:
it is hiding Shorts videos at https://www.youtube.com/feed/subscriptions

Reply:
This toggle is hiding shorts videos at https://www.youtube.com/feed/subscriptions

Reply:
Done

Reply:
now i figured you had just removed one in between /by mistake